### ISSUE TITLE
SA1027 Should Not Warn for Commented Out Code

### ISSUE THREAD

**Issue by jodihamann** [DotNetAnalyzers/StyleCopAnalyzers#2794]
When ```useTabs``` is set to ```true```, using Ctrl-K Ctrl-C to comment out multiple lines of code with different levels of indentation creates lines which violate SA1027.

For example:
![multiline_statement](https://user-images.githubusercontent.com/7935258/47115986-528b7580-d22e-11e8-80f2-8795e6e0eede.png)
Becomes:
![image](https://user-images.githubusercontent.com/7935258/47116064-81095080-d22e-11e8-80da-318be06ea104.png)

This gives an error because the comment includes the tab for the more indented line, and because StyleCop gives warnings for tabs used in single-line comments (```//```).

SA1027 currently only ignores lines which begin with four slashes (```////````).

The relevant lines of code from SA1027UseTabsCorrectly.cs:
```
if (trivia.ToString().StartsWith("////"))
{
    // Exclude comments starting with //// because they could contain commented code which contains
    // string or character literals, and we don't want to change the contents of those strings.
    builder.Add(trivia.Span);
}
```

The unit tests in SA1027UseTabsUnitTests.cs (specifically TestInvalidTabsInCommentsAsync) show that this is the expected behavior, i.e. treat tabs after two slash comments as incorrect, but ignore tabs after four slashes.

Is there a reason for the different treatment of single-line comments?

**Comment by sharwell** [comment]
> Is there a reason for the different treatment of single-line comments?

StyleCop Classic defined the original behavior (four slashes) for identifying commented code. We carried over the behavior here.
